packages/utilities/src/rtl.ts

```diff
@@ -61,3 +60,22 @@ export function getRTLSafeKeyCode(key: number): number {
 
   return key;
 }
+
+/**
+ * Helper utility that gets dir attribute from document elements.
+ * @returns The RTL state of the document (true if in RTL.)
+ */
+export function getDocumentRTL(): boolean {
+  let doc = getDocument();
```

Comment:
As `doc` doesn't change throughout the function, could we use `const` instead of `let`?

Reply:
Yeah. This evolved from a prior impl that needed a let.